requirements_all.txt

```diff
@@ -1328,7 +1328,7 @@ mopeka-iot-ble==0.7.0
 motionblinds==0.6.23
 
 # homeassistant.components.motionblinds_ble
-motionblindsble==0.0.8
+motionblindsble==0.0.9
```

Comment:
Please bump in separate PR

Reply:
Without the bump the battery sensor won't work properly though as the callback has been changed?

Reply:
I am not saying that that PR should be merged later. That bump should be merged first before adding this feature